prefect-workflows.md

```diff
@@ -0,0 +1,103 @@
+deploying parametrized machine learning models, ad-hoc reporting, and
+handling event-driven processes.
+
+Typically this means developers need a workflow system which can do things such as:
+- retry failed tasks
+- schedule jobs to run automatically
+- log detailed progress (and history) of the workflow
+- provide a dashboard / UI for inspecting system health
+- provide notification hooks for when things go wrong
+
+among many other things.  We at Prefect like to think of a workflow system as
+a technical insurance policy - you shouldn't really notice it much when 
+things are going well, but it should be maximally useful when things go wrong.
+
+Prefect's goal is to build the next generation workflow system.  Older systems
+such as [Airflow](https://medium.com/the-prefect-blog/why-not-airflow-4cfa423299c4) and Luigi are limited 
+by their model of workflows as slow-moving, regularly scheduled,
+with limited inter-task communication.  Prefect, on the other hand, embraces
+this new reality and makes very few assumptions about the nature and requirements of
+workflows, thereby supporting more dynamic use cases in both data engineering
+and data science.
+
+
+How Dask helps
+--------------
+
+Prefect was designed and built with Dask in mind.  Historically, workflow systems
+such as [Airflow](https://airflow.apache.org/) handled _all_ scheduling, of both
+workflows _and_ the individual tasks contained within the workflows. This pattern introduces a number of problems:
+- this puts an enormous burden on the central scheduler (it is scheduling _every single action_ taken in the system)
+- it adds non-trivial latency to task runs
+- in practice, this limits the amount of dynamicism workflows can have
+- it also tends to limit the amount of data tasks can share, as all information is routed through the central scheduler
+- it requires users to have an external scheduler service running to run their workflows at all!
+
+Instead, Prefect handles the scheduling of _workflows_, and lets Dask
+handle the scheduling and resource management of _tasks_ within each workflow.  This
+provides a number of benefits out of the box:
+
+- **Task scheduling:** Dask handles all task scheduling within a workflow, allowing Prefect to incentivize smaller tasks which Dask schedules with millisecond latency
+- **"Dataflow":** because Dask handles serializing and communicating the appropriate information between Tasks, Prefect can support "dataflow" as a first-class pattern
+- **Distributed computation:** Dask handles allocating Tasks to workers in a cluster, allowing users to immediately realize the benefits of distributed computation with minimal overhead
+- **Parallelism:** whether running in a cluster or locally, Dask provides parallel Task execution off the shelf
+
+Additionally, because Dask is written in pure Python and has an active open source community,
+we can very easily get feedback on possible bugs, and even contribute to improving the software ourselves.
```

Comment:
Thoughts on pointing readers to the [futures](https://docs.dask.org/en/latest/futures.html) docs?  Someone reading this and identifying with your experience may want to know where to start with the project.  That seems like the natural spot.

Reply:
Yea absolutely - will update.

Reply:
Updated a few lines above this one, let me know if that's what you had in mind